app/config.js

```diff
@@ -8,7 +8,9 @@ const notify = require('./notify');
 const Config = require('electron-config');
 
 // local storage
-const cache = new Config();
+const winCfg = new Config({name: 'winConf'});
+winCfg.defaultPosition = [50, 50];
+winCfg.defaultSize = [540, 380];
```

Comment:
That's not what I meant by defaults. See: https://github.com/sindresorhus/electron-config#defaults